packages/main/src/plugin/skill/skill-manager.spec.ts

```
@@ -697,26 +701,10 @@ test('listSkillFolders should include built-in kaiden folder after init', async
-        ReturnType<typeof readdir>
-      >;
-    }
-    return [] as unknown as Awaited<ReturnType<typeof readdir>>;
-  });
-  vi.mocked(readFile).mockImplementation(async (p: unknown) => {
-    const path = String(p);
-    if (path === join(EXTRA_DIR, 'extra-skill', SKILL_FILE_NAME)) return EXTRA_SKILL_MD;
-    throw new Error(`File not found: ${path}`);
-  });
+  mockDirectoryWithSkills(
+    [{ name: 'extra-skill', content: `---\nname: extra-skill\ndescription: An extra skill\n---\n# Extra\n` }],
```

Biome (2.4.10)

[error] Do not use template literals if interpolation and special-character handling are not needed.

(lint/style/noUnusedTemplateLiteral)